packages/provider/rpc.ts

```diff
@@ -10,7 +10,8 @@ export default {
     eth: {
       accounts: "eth_accounts",
       blockNumber: "eth_blockNumber",
-      chainId: "eth_chainId"
+      chainId: "eth_chainId",
+      waalletConnect: "eth_waalletConnect"
```

Comment:
I think it should not be a rpc method. It would be better to remove modifications for provider, and keep only those code demonstrating how to open popup